action-templates/actions/action-maven-release/action.yml

```diff
@@ -5,46 +5,46 @@ inputs:
     required: false
     default: 21
     type: string
-    description: configure the java version
+    description: "Configure java version"
   app-path:
     required: true
     type: string
-    description: path where the pom.xml is
-  releaseVersion:
+    description: "Path to pom.xml"
+  release-version:
     required: true
     type: string
-    description: version which will be released
-  developmentVersion:
+    description: "Version which will be released"
+  development-version:
     required: true
     type: string
-    description: next version with snapshot
-  skipDeployment:
+    description: "Next snapshot version"
+  skip-deployment:
     default: true
     type: boolean
-    description: skip deployment to maven central
+    description: "Skip deployment to Maven Central"
   SIGN_KEY_PASS:
     required: true
     type: string
-    description: env variable for GPG private key passphrase
+    description: "GPG private key passphrase"
   CENTRAL_USERNAME:
     required: true
     type: string
-    description: env variable for username in deploy
+    description: "Maven Central username"
   CENTRAL_PASSWORD:
     required: true
     type: string
-    description: env variable for token in deploy
+    description: "Maven Central token"
   GPG_PRIVATE_KEY:
     required: true
     type: string
-    description: Value of the GPG private key to import
+    description: "GPG private key to import"
 
 outputs:
   MVN_ARTIFACT_ID:
```

Comment:
wollen wir die output Variablen wie die input-Variablen in Klein-Buchstaben mit minus getrennt haben (artifact-name) oder wie hier mit Großbuchstaben und _ (MAVEN_ARTIFACT_ID). Ein Mischmasch sollten wir vermeiden..